pkg/pool-manager/virtualmachine_pool.go

```diff
@@ -143,6 +144,68 @@ func (p *PoolManager) allocateRequestedVirtualMachineInterfaceMac(requestedMac s
 	return nil
 }
 
+func (p *PoolManager) initVirtualMachineMap() error {
```

Comment:
I don't see this function called. It should be used in the same commit you introduce it. You can merge it with the following commit.

Reply:
Done